python/modules/IcePy/Logger.cpp

```diff
@@ -414,7 +414,7 @@ IcePy::cleanupLogger()
     //
     // Python is about to exit; we need to remove the wrapper around the process logger.
     //
-    Ice::setProcessLogger(0);
+    Ice::setProcessLogger(nullptr);
```

Comment:
That is the only thing done by the cleanup, not clear we need it at all.

Reply:
I don't think so. At this point we are unloading the extension nothing should be using the process logger.